src/vs/workbench/api/common/extHostSpeech.ts

```diff
@@ -64,7 +68,11 @@ export class ExtHostSpeech implements ExtHostSpeechShape {
 		const cts = new CancellationTokenSource();
 		this.sessions.set(session, cts);
 
-		const textToSpeech = disposables.add(provider.provideTextToSpeechSession(cts.token));
+		const textToSpeech = await provider.provideTextToSpeechSession(cts.token);
```

Comment:
does this still need to be added to disposables?

Reply:
The session is no longer disposable, thats part of the PR.